humble/distribution.yaml

```diff
@@ -9781,6 +9781,19 @@ repositories:
       url: https://github.com/pal-robotics/talos_robot.git
       version: humble-devel
     status: developed
+  talos_simulation:
+    release:
+      packages:
+      - talos_gazebo
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/pal-gbp/talos_simulation-release.git
```

Comment:
This is OK for release into Humble, but if you ever want to release into Rolling this would have to be moved into the `ros2-gbp` organization.

Reply:
Sure. Thanks for letting us know. :)